ghost/i18n/locales/sk/comments.json

```diff
@@ -10,42 +10,42 @@
     "Add reply": "Pridať odpoveď",
     "Already a member?": "Už ste členom?",
     "Anonymous": "Anonymne",
-    "Are you sure?": "",
+    "Are you sure?": "Ste si istý?",
     "Become a member of {{publication}} to start commenting.": "Staňte sa členom {{publication}}, aby ste mohli začať komentovať",
     "Become a paid member of {{publication}} to start commenting.": "Staňte sa plateným členom {{publication}}, aby ste mohli začať komentovať",
     "Cancel": "Zrušiť",
     "Comment": "Komentár",
     "Complete your profile": "Dokončite váš profil",
     "Delete": "Vymazať",
-    "Deleted": "",
+    "Deleted": "Vymazané",
     "Deleted member": "Vymazaný člen",
-    "Deleting": "",
+    "Deleting": "Vymazávam",
     "Discussion": "Diskusia",
     "Edit": "Upraviť",
-    "Edit this comment": "Upraviť tento komentár",
+    "Edit this comment": "Upraviť komentár",
     "edited": "upravený",
     "Enter your name": "Zadajte vaše meno",
     "Expertise": "Odbornosť",
     "Founder @ Acme Inc": "Zakladateľ @ Acme Inc",
     "Full-time parent": "Rodič na plný úväzok",
     "Head of Marketing at Acme, Inc": "Vedúci marketingu v spoločnosti Acme, Inc",
-    "Hidden for members": "",
+    "Hidden for members": "Skryté pre členov",
```

Comment:
AI suggests Skryté pred členmi, meaning that members cannot see the comment.

Reply:
Skryté pre členov is ok